Frends.Files.Copy/CHANGELOG.md

```
@@ -1,5 +1,11 @@
 # Changelog
 
+# [1.2.0] - 2025-04-08
+### Added
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)